tests/git_bisect_toaster.sh

```diff
@@ -0,0 +1,81 @@
+#!/bin/bash
+
+# so we can stop if we need to
+function trap_exit {
+    cd $POKYDIR
+    echo
+    echo "Exiting due to signal..."
+    # this is messy and heavy handed as it will kill other bisects running at the
+    # same time.  Unfortunately, bisect/runtests was quite incorrigible.
+    ps augxww | grep "git-bisect run" | grep runtests.sh | awk '{print $2}' | \
```

Comment:
If you could save the pid of "git-bisect run", then you could find the runtests.sh instance of this instance only. Then forward the signal to rntests.sh. It should exit gracefully.